tools/buildmgr/cbuildgen/installer/install.sh

```diff
@@ -150,6 +151,7 @@ case $OS in
     compiler6_default_path=$(wslpath "${programfiles}/ARMCompiler6.16/bin")
     compiler5_default_path=$(wslpath "${programfiles} (x86)/ARM_Compiler_5.06u7/bin")
     gcc_default_path=$(wslpath "${programfiles} (x86)/GNU Arm Embedded Toolchain/10 2020-q4-major/bin")
+    iar_default_path=$(wslpath "${PROGRAMFILES} (x86)/IAR Systems/Embedded Workbench 8.4/arm/bin")
```

Comment:
I guess it should be lowercase?
```suggestion
    iar_default_path=$(wslpath "${programfiles} (x86)/IAR Systems/Embedded Workbench 8.4/arm/bin")
```